cura/MultiplyObjectsJob.py

```diff
@@ -93,17 +73,28 @@ def run(self) -> None:
                     child.callDecoration("setBuildPlateNumber", build_plate_number)
 
                 nodes.append(new_node)
-                current_progress += 1
-                status_message.setProgress((current_progress / total_progress) * 100)
-                QCoreApplication.processEvents()
-                Job.yieldThread()
-            QCoreApplication.processEvents()
-            Job.yieldThread()
-
+        factor = 10000
+        found_solution_for_all, node_items = findNodePlacement(nodes, Application.getInstance().getBuildVolume(), fixed_nodes, factor = 10000)
+        not_fit_count = 0
         if nodes:
             operation = GroupedOperation()
-            for new_node in nodes:
-                operation.addOperation(AddSceneNodeOperation(new_node, current_node.getParent()))
+            for new_node, node_item in zip(nodes, node_items):
+                operation.addOperation(AddSceneNodeOperation(new_node, root))
+
+                if node_item.binId() == 0:
+                    # We found a spot for it
+                    rotation_matrix = Matrix()
+                    rotation_matrix.setByRotationAxis(node_item.rotation(), Vector(0, -1, 0))
+                    operation.addOperation(RotateOperation(new_node, Quaternion.fromMatrix(rotation_matrix)))
+                    operation.addOperation(
+                        TranslateOperation(new_node, Vector(node_item.translation().x() / factor, 0,
+                                                        node_item.translation().y() / factor)))
+                else:
+                    # We didn't find a spot
+                    operation.addOperation(
+                        TranslateOperation(new_node, Vector(200, 0, -not_fit_count * 20), set_position=True))
+                    not_fit_count += 1
+
```

Comment:
Call Libnest2dArrange.arrange instead of lines 77-97 in MultiplyObjectsJob.py as these lines seems to be doing the exact same thing